explainer.md

```diff
@@ -144,29 +147,32 @@ suffer from similar drawbacks.
 - Measuring layout, with intent of sizing containers without actually displaying
   the contents.
 
-In general, large-scale updates to application state in a web app can cause
-updates which induce large document lifecycle updates, including style, layout,
+In general, **large-scale updates** to application state in a web app can cause
+updates which **induce large document lifecycle updates**, including style, layout,
 compositing, and paint. In turn, these can cause jank on the page, due to
 lifecycle updates being synchronous with script and user interactions.
 
+In a large-enough web app, even having small updates might be expensive even though
```

Comment:
I find this sentence confusing. Are you trying to simply say that if the document is very large, even small diffs to it can be expensive?